azure-pipelines.yml

```diff
@@ -14,7 +14,6 @@ jobs:
       sudo apt-get install              \
           intel-oneapi-common-vars      \
           intel-oneapi-common-licensing \
-          intel-oneapi-tbb-devel        \
```

Comment:
I guess we don't need TBB-devel, but we need tbb for runtime to actually execute CPU threaded stuff?

Reply:
Yes, tbb is runtime dependency of threaded DAAL.